modern_treasury/modern_treasury.py

```diff
@@ -9,9 +9,11 @@
 from modern_treasury.objects.request.expected_payment import ExpectedPaymentRequest
```

Comment:
I would add another method to be able to choose the connection we want.
Here is an example response from the connections list:

```
[{"id":"289cd2f0-20bf-4f73-98ae-1535f90f31e0","object":"connection","live_mode":false,"vendor_id":"example1","vendor_name":"Gringotts Wizarding Bank","vendor_customer_id":null,"discarded_at":null,"created_at":"2021-05-19T14:39:28Z","updated_at":"2021-05-19T14:39:28Z"},{"id":"55a000f7-6eb1-42ff-a160-6ed3c1087b60","object":"connection","live_mode":false,"vendor_id":"example2","vendor_name":"Iron Bank of Braavos","vendor_customer_id":null,"discarded_at":null,"created_at":"2021-05-19T14:39:28Z","updated_at":"2021-05-19T14:39:28Z"}]```

We should be able to grab the connection based on the vendor_name and vendor_id